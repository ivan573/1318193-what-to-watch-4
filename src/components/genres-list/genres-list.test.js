import React from "react";
import renderer from "react-test-renderer";
import GenresList from "./genres-list.jsx";

test(`Genres list matches snapshot`, () => {
  const tree = renderer
    .create(<GenresList
      genres={[`All genres`, `Kids & Family`, `Drama`, `Comedy`]}
      activeGenre={`All genres`}
      onGenreClick={() => {}}
      changeActiveItem={() => {}}
    />).toJSON();

  expect(tree).toMatchSnapshot();
});
