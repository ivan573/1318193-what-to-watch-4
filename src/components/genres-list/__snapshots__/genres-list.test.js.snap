// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Genres list matches snapshot 1`] = `[Function]`;
